src/server/migrations/20211105095000_products-table.js

```diff
@@ -0,0 +1,30 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('products', (table) => {
+    table.increments('id').primary();
+    table.string('name').notNullable();
+    table.decimal('price').notNullable();
+    table
+      .enum('color', ['red', 'green', 'blue', 'yellow', 'white', 'black'])
+      .notNullable();
+    table
+      .enum('size', ['small', 'medium', 'large', 'extra-large', 'doubleXL'])
+      .notNullable();
+    table
+      .enum('status', ['out_of_stock', 'in_stock', 'running_low'])
+      .notNullable();
+    table.datetime('created_at').defaultTo(knex.fn.now()).notNullable();
+    table.binary('picture').notNullable();
```

Comment:
@dpfernandes if we just storing link to local assets folder (as I did in seeding PR) than this can be string I guess, can you have a look

Reply:
Yes i used string before, since we will store the link for the images, but later i just followed the correction from the code since it was not working with foreign key constraint. And i just forgot to see for picture datatype.
Anyway, i will correct it soon. Thanks Julia!

Reply:
Maybe i can merge the PR now right? @Jul-S

Reply:
@BikramRawat as said in readme - Every pull request needs 2 approvals to be merged. One of them must be from a mentor.